comps/asr/src/integrations/dependency/whisper/Dockerfile

```diff
@@ -20,16 +20,16 @@ COPY --chown=user:user comps /home/user/comps
 USER user
```

Comment:
Should we put the `dependecy` folder to `comps/asr/src/dependency` or just here? @lvliang-intel

Reply:
yes